packages/thirdweb/src/react/core/hooks/transaction/usePrepareTransaction.ts

```diff
@@ -172,23 +172,27 @@
       ],
       queryFn: async () => {
         const preparedTx = prepareContractCall(extensionOrTxOptions);
-        return await toSerializableTransaction({
-          transaction: preparedTx,
-          from: options?.from,
-        });
+        try {
+          return await toSerializableTransaction({
+            transaction: preparedTx,
+            from: options?.from,
+          });
+        } catch (e) {
+          console.log("ERROR", e);
```

Comment:
Why do we catch here? Don't we wanna throw so that the use query propagates the error?

Reply:
Ah shoot, that was from debugging thank you for catching that